### PR TITLE
WIP: Translate contribute/intermediate in french

### PR DESCRIPTION
**Pull request by AWKIF**

### CODE REVIEW

File: content/fr/docs/contribute/intermediate.md [hunk 1]
```diff
@@ -0,0 +1,834 @@
+---
+title: Contribution intermédiaire
+slug: intermediaire
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Cette page assume que vous avez lu et compris les tâches dans le topic
```
Comment (perriea) [inline content/fr/docs/contribute/intermediate.md]: assume ? Change the verb please

File: content/fr/docs/contribute/intermediate.md [hunk 2]
```diff
@@ -0,0 +1,834 @@
...
+{{% capture body %}}
+
+Maintenant que vous vous êtes fait la main et que vous avez aidé à la documentation de Kubernetes en
+suivant les principes du topic [commencez à contribuer](/docs/contribute/start/),
+vous êtes surement prêt pour en faire plus. Ces tâches assument que vous avez, ou voulez
+améliorer vos connaissances sur ces différents domaines:
+
+- Les concepts de Kubernetes
+- Kubernetes documentation workflows
+- Where and how to find information about upcoming Kubernetes features
```
Comment (perriea) [inline content/fr/docs/contribute/intermediate.md]: delete it, this sentence is translated below

File: content/fr/docs/contribute/intermediate.md (hunk 1 above)
Comment (AWKIF) [reply content/fr/docs/contribute/intermediate.md]: "assume" is a french verb but "suppose" works better thanks

File: content/fr/docs/contribute/intermediate.md (hunk 2 above)
Comment (AWKIF) [reply content/fr/docs/contribute/intermediate.md]: done

**Comment by perriea** [comment]
Hi @AWKIF, do not hesitate to use amend. It allows to add changes in your last commit.
ex: `git commit --amend`

**Comment by sftim** [comment]
How's this going @AWKIF?

**Comment by AWKIF** [comment]
i think this has already been done in another PR, will update it tomorrow

**Pull request closed without merging**